cli/cmd/vuln_host_test.go

```diff
@@ -0,0 +1,97 @@
+//
+// Author:: Darren Murray (<dmurray-lacework@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"github.com/lacework/go-sdk/api"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestListCvesFilterSeverity(t *testing.T) {
+	vulCmdState.Severity = "critical"
+	mockCves := []api.HostVulnCVE{mockCveOne}
+	result, output := hostVulnCVEsTable(mockCves)
+
+	assert.Equal(t, len(result), 1)
+	assert.Equal(t, output, "\n 1 of 2 cve(s) showing \n")
+
+	clearVulnFilters()
```

Comment:
After setting `vulCmdState.Severity` to critical, defer this clear function.

Reply:
👍